mcmc/MinuitFit.cpp

```diff
@@ -46,7 +46,7 @@ void MinuitFit::runFit() {
       for(int i = 0; i < (*it)->getSize(); ++i, ++ParCounter)
       {
           //KS: Index, name, prior, step scale [differrent to MCMC],
-          minuit->SetVariable(ParCounter, ((*it)->getParName(i)), (*it)->getParInit(i), (*it)->getDiagonalError(i)/10);
+          minuit->SetVariable(ParCounter, ((*it)->GetParName(i)), (*it)->getParInit(i), (*it)->getDiagonalError(i)/10);
```

Comment:
Hyper nitpicky, would you be able to change the covarianceBase getter to `getParName` instead to make method naming consistent?

Reply:
I only changed this to make code compile 😅

Reply:
That is fair lol 😅